lib/hanami/model/migrator.rb

```diff
@@ -237,9 +237,9 @@ def self.version
       #
       # @since 0.7.0
       # @api private
-      def initialize(configuration: self.class.configuration)
+      def initialize(configuration: self.class.configuration, stream: $stdout)
```

Comment:
@pascalbetz @mereghost TBH, I injected that dependency only for testing purposes, but I don't like it. Also, look at how many times is passed down from method call, to method call, deep in the stack.

I can refactor it, by adding a `Configuration#migrations_logger` as private API, so we can remove it from all these method signatures **and** have an easy access to stub it during test.